### DISCUSSION TITLE
Why doesn't allow use Chinese in username of consumer?

### DISCUSSION THREAD

**Discussion by iSuperCoder** [apache/apisix#5907] [General]
为什么不允许消费者的用户名里包含中文呢？

**Comment by juzhiyuan** [comment]
Hi, here is the Consumer's username field RegExp https://github.com/apache/apisix/blob/master/apisix/schema_def.lua#L693

![image](https://user-images.githubusercontent.com/2106987/147621232-80633c0a-70b0-4fc1-91b3-b9712a1e05e2.png)

As for why it doesn't allow Chinese characters, I'm not sure 🤔
